nemo/collections/tts/models/uniglow.py

```diff
@@ -21,7 +21,7 @@
 from pystoi import stoi
 from pytorch_lightning.loggers import LoggerCollection, TensorBoardLogger
 
-from nemo.collections.tts.helpers.helpers import OperationMode, waveglow_log_to_tb_func
+from nemo.collections.tts.helpers.helpers import OperationMode, split_view, waveglow_log_to_tb_func
```

Comment:
I don't see `split_view` in `main`'s helpers.py. @ericharper do you know why helpers.py isn't being updated?

Reply:
It was lost when merging, We can add back it in.